README.md

```diff
@@ -5,7 +5,7 @@
 Task:
 
 There are some figures of the following types: square, rectangle, right triangle, circle, isosceles trapezoid.
-You need to create corresponding classes for them(`Square`, `Rectangle`, `RightTriangle`, `Circle`, `IsoscelesTrapezoid`)
+You need to create corresponding classes for them(`core.basesyntax.Square`, `core.basesyntax.Rectangle`, `core.basesyntax.RightTriangle`, `core.basesyntax.Circle`, `core.basesyntax.IsoscelesTrapezoid`)
 
```

Comment:
1. be careful with project-wide replacements
2. review the files you are about to commit
3. please undo unintended changes

Reply:
i fixed my previous commit where it was accidentally replaced with intelij refactor